eq-author/src/components/Panel/ErrorPanel.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled, { css } from "styled-components";
+import PropTypes from "prop-types";
+import { colors } from "constants/theme";
+
+const destinationPanel = css`
+  margin-left: 41.7%;
+`;
+
+const logicPanel = css`
+  margin-left: 12.5%;
+`;
+
+const confirmationPanel = css`
+  margin-left: 3%;
+`;
+
+const ErrorPanel = ({ variant, children }) => {
+  const StyledPanel = styled.div`
+    background-color: ${colors.errorSecondary};
+    border: 0;
+    border-radius: 0;
+    border-left: 0.5em solid ${colors.errorPrimary};
+    margin: 1em 0;
+    display: inline-block;
+    width: auto;
+
+    ${variant === "destination" && destinationPanel};
+    ${variant === "logic" && logicPanel};
+    ${variant === "confirmation" && confirmationPanel};
+  `;
+
+  return <StyledPanel>{children}</StyledPanel>;
+};
```

Comment:
```suggestion
  const StyledPanel = styled.div`
    background-color: ${colors.errorSecondary};
    border: 0;
    border-radius: 0;
    border-left: 0.5em solid ${colors.errorPrimary};
    margin: 1em 0;
    display: inline-block;
    width: auto;

    ${({variant}) === "destination" && destinationPanel};
    ${({variant}) === "logic" && logicPanel};
    ${({variant}) === "confirmation" && confirmationPanel};
  `;
  
const ErrorPanel = ({ variant, children }) => <StyledPanel variant={variant}>{children}</StyledPanel>;
```

It's better practice to always define styled components outside the ones they're used in as sometimes it can lead to strange re-rendering patterns as data changes.